sample_onsr/oci-objectstorage-onsr-put-object-java/func.yaml

```diff
@@ -0,0 +1,8 @@
+schema_version: 20180708
+name: oci-objectstorage-onsr-put-object-java
+version: 0.0.17
+runtime: java
+build_image: fnproject/fn-java-fdk-build:jdk11-1.0.130
+run_image: fnproject/fn-java-fdk:jre11-1.0.130
+cmd: com.example.fn.ObjectStorageOnsrPutObject::handle
+timeout: 120
```

Comment:
Time out is added here but not in go example, we should try to make examples consistent.
Check if we should provide the shape too.